data/showcase/leximpact_impot_revenu.yml

```diff
@@ -0,0 +1,14 @@
+title: Simulateur de réformes de l'impôt sur le revenu - LexImpact
+description:
+  fr: LexImpact utilise OpenFisca et son API Python pour simuler l'impact des amendements paramétriques sur les foyers fiscaux, grâce à un simulateur dédié à l'impôt sur le revenu.
+  en: LexImpact uses OpenFisca and its Python API to evaluate the impact of parametric amendments on tax households, thanks to an income revenue tax simulator.
+package: france
+website: http://leximpact.an.fr/ir  # https://leximpact.beta.gouv.fr in 2019
+until: 2023-01-16
```

Comment:
Became `http://leximpact.an.fr/ir` in 2020 and was merged into `leximpact_socio_fiscal` showcase in 2023. Do we want to tell this story? 😅

Reply:
No, I don't think that story there is relevant 🙂 we mostly want to demonstrate the adoption of OpenFisca.

Reply:
The `until` is there to tell that the product was retired at some point; I don't think that's the case of this reuse, is it?

Reply:
If the name is ambiguous to you, we could rename that property into something like `retired_on`

Reply:
As this PR is, you marked `leximpact_impot_revenu` as disused (by adding an `until` entry). Is that really what you wanted?
![image](https://user-images.githubusercontent.com/222463/217551774-18d16eaf-fc40-405a-85c1-47865bcf7ef2.png)

Reply:
> The `until` is there to tell that the product was retired at some point; I don't think that's the case of this reuse, is it?

It is since January. There is now a redirection on http://leximpact.an.fr/ir to the socio-fiscal application.

Reply:
Okay, thank you for this clarification! I am not sure I fully understand the point of having two entries then.